sample/src/test/kotlin/com/slack/circuit/sample/PresenterExtensions.kt

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright (C) 2022 Slack Technologies, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.slack.circuit.sample
+
+import app.cash.molecule.RecompositionClock
+import app.cash.molecule.moleculeFlow
+import app.cash.turbine.ReceiveTurbine
+import app.cash.turbine.test
+import com.slack.circuit.Presenter
+import kotlinx.coroutines.flow.Flow
+import kotlinx.coroutines.flow.emptyFlow
+
+suspend fun <State : Any, Event : Any> Presenter<State, Event>.test(
+  block: suspend ReceiveTurbine<State>.() -> Unit
+) = test(emptyFlow(), block)
+
+suspend fun <State : Any, Event : Any> Presenter<State, Event>.test(
```

Comment:
For now I put this under the sample project. Do we want to include this as part of circuit? Maybe in a standalone test project?

Reply:
It assumes users of circuit will want to use molecule + turbine for testing

Reply:
yeah eventually we'll want to extract a test-fixtures of some sort for these 👍